wiki/People/Quality_Assurance_Team/en.md

```diff
@@ -8,7 +8,7 @@ The **Quality Assurance Team**, commonly referred to as QAT, are the judges of q
 
 QAT members will be held to strict activity standards as their role requires them to be regularly involved in assessing beatmap quality.
 
-Members of this team will be promoted from [BNs](/wiki/People/Beatmap_Nomination_Group), should they accept the new role offered. Members of QAT have a red name in in-game chat and on the forums, whereas BNs no longer do.
+Members of this team will be promoted from [BNs](/wiki/People/Beatmap_Nomination_Group/), should they accept the new role offered. Members of QAT have a red name in in-game chat and on the forums, whereas BNs only have a purple name on the forum.
```

Comment:
what I meant to say was to remove `en.md` the it's good to go
~~(stupid mobile making me view commit instead of pull request)~~